### PR TITLE
Add parameter prompt for scheduler show --status ref #4405

### LINKED ISSUE #4405: Add parameter prompt for scheduler show --status
There is no prompt for `scheduler show --status` command to indicate the value of status. See:

```
» scheduler show --status
Usage:
  pd-ctl scheduler show [flags]

Flags:
  -h, --help            help for show
      --status string   the scheduler status

flag needs an argument: --status
```

### PR DESCRIPTION
**Pull request by Dagon0577**
close #4405

### What problem does this PR solve?
Add parameter prompt for scheduler show --status ref #4405

### What is changed and how it works?
Just add parameter prompt for scheduler show --status
### Check List

Tests

- Unit test
- Integration test
### Release note

```release-note
None.
```

### CODE REVIEW

**Comment by rleungx** [comment]
Perhaps we need another status which could be named "running" or "enabled" and make the default output of `scheduler show` command to list all running schedulers. /cc @nolouch

**Comment by JayiceZ** [comment]
> Perhaps we need another status which could be named "running" or "enabled" and make the default output of `scheduler show` command to list all running schedulers. /cc @nolouch

But it seems like if the status is meaningless, then aall running schedulers will be output by default

see https://github.com/tikv/pd/blob/eba75c3080a0db2d5edda7a2a8628e3a08c1af87/server/api/scheduler.go#L94

**Comment by rleungx** [comment]
> > Perhaps we need another status which could be named "running" or "enabled" and make the default output of `scheduler show` command to list all running schedulers. /cc @nolouch
> 
> But it seems like if the status is meaningless, then all running schedulers will be output by default
> 
> see
> 
> https://github.com/tikv/pd/blob/eba75c3080a0db2d5edda7a2a8628e3a08c1af87/server/api/scheduler.go#L94

It will actually print all existing schedulers, but we cannot tell if it is running.

**Review by disksing — APPROVED** [review]

**Comment by Dagon0577** [comment]
> 

maybe you can do it, i will close this

File: tools/pd-ctl/pdctl/command/scheduler.go [hunk 1]
```diff
@@ -87,7 +87,7 @@ func NewShowSchedulerCommand() *cobra.Command {
 		Short: "show schedulers",
 		Run:   showSchedulerCommandFunc,
 	}
-	c.Flags().String("status", "", "the scheduler status")
+	c.Flags().String("status", "", "the scheduler status value can be [paused | disabled]")
```
Comment (bufferflies) [inline tools/pd-ctl/pdctl/command/scheduler.go:90]: the paused and disbaled will support all or one should be deprecated.

**Comment by nolouch** [comment]
> Perhaps we need another status which could be named "running" or "enabled" and make the default output of `scheduler show` command to list all running schedulers. /cc @nolouch

It's good point, we can do it in another PR or issues.

**Review by nolouch — APPROVED** [review]
lgtm

**Pull request merged**